helm/hpcc/values.yaml

```diff
@@ -300,14 +301,18 @@ sasha:
     #throttle: 0 # throttle ratio (0-99, 0 no throttling, 50 is half speed)
 
   dfurecovery-archiver:
-    #disabled: true
+    disabled: true
+    service:
```

Comment:
I assume not intended, just disabled while testing?

Reply:
reverted